packages/console/src/pages/Connectors/components/ConfigForm/index.tsx

```diff
@@ -64,7 +65,7 @@ const ConfigForm = ({ formItems }: Props) => {
     }
 
     if (item.type === ConnectorConfigFormItemType.Number) {
-      return <TextInput type="number" {...buildCommonProperties()} />;
+      return <TextInput {...buildCommonProperties({ valueAsNumber: true })} />;
```

Comment:
do we have a convention to determine which kind of number input should not have arrows?